feature-utils/silly-i18n/rollup-plugin.js

```diff
@@ -0,0 +1,57 @@
+import fs from "fs";
```

Comment:
Have you considered something different from the root dir? It's better kept for configuration and metadata.

Reply:
Why is this file at the root directory for the module?

Reply:
Ok, I finally realized you are not actually talking about this line of code but about where the Rollup plugin is located in the source tree. Where do you suggest I put it?

Reply:
Well, I don't really think at this point a rollup plugin is the best solution. If the solution is finally a script, those generally go to a `bin` directory. If it's finally a plugin, follow the same rules as `rollup-plugin-copy-watch`, essentially.

Reply:
Discussion is elsewhere, but consistent would be `silly-i18n/rollup-plugin-gen-i18n` or something.

Reply:
Not sure if it makese sense to copy the `gen` part since  `@polypoly-eu/podjs/rollup-plugin-gen-podjs/genPodjs.js` generates (or rather copies) a file (i.e. `pod.js`) while this plugin handles a particular kind of import.